tests/integration/test_elasticsearch.py

```diff
@@ -66,7 +67,14 @@ def test_domain_creation():
     status = es_client.describe_elasticsearch_domain(DomainName=TEST_DOMAIN_NAME)
     assert_equal(status['DomainStatus']['DomainName'], TEST_DOMAIN_NAME)
     assert_true(status['DomainStatus']['Created'])
+    assert_false(status['DomainStatus']['Processing'])
     assert_false(status['DomainStatus']['Deleted'])
+    assert_equal(status['DomainStatus']['Endpoint'], TEST_ENDPOINT_URL)
```

Comment:
The tests in Travis-CI are currently failing at this line. Please also use `aws_stack.get_elasticsearch_endpoint()` here and remove the `TEST_ENDPOINT_URL` variable.